docs/install/create-project-classic-ui.md

```diff
@@ -0,0 +1,224 @@
+---
+myst:
+  html_meta:
+    "description": "Create a Plone project with Classic UI (stable release)"
+    "property=og:description": "Create a Plone project with Classic UI (stable release)"
+    "property=og:title": "Create a Plone project with Classic UI (stable release)"
+    "keywords": "Plone, Plone 6, Classic UI, create, project, install, cookiecutter"
+---
+
+
+(create-a-project-classic-ui-label)=
+
+# Create a project with Classic UI (stable release)
+
+This chapter describes how you can create a web application using the current **stable release** version of Plone with **Classic UI** for the frontend, while having full control over its development and deployment.
+
+```{seealso}
+For other installation options, see {doc}`/install/index`.
+```
+
+
+## System requirements
+
+Plone 6 has both hardware requirements and software pre-requisites.
```

Comment:
@stevepiercy Why do we prefer prerequisite with a hyphen? From what I can see it's in the dictionary without one.

Reply:
I created a new issue to handle this. It's everywhere. https://github.com/plone/documentation/issues/1720